staging/src/k8s.io/apiserver/pkg/util/flowcontrol/controller_test.go

```diff
@@ -227,7 +227,7 @@ var mandQueueSetNames, exclQueueSetNames = func() (sets.String, sets.String) {
 
 func TestConfigConsumer(t *testing.T) {
 	rngOuter := rand.New(rand.NewSource(1234567890123456789))
-	for i := 1; i <= 20; i++ {
+	for i := 1; i <= 10; i++ {
 		rng := rand.New(rand.NewSource(int64(rngOuter.Uint64())))
```

Comment:
We also need to reduce the logging produced by this package by default. In addition to the number of iterations, excessive logging is part of why this package takes so long to test, and bloats test output enormously.

commenting out the Log statements, the package runs ~3x faster:
```
time $(go test -race -v k8s.io/apiserver/pkg/util/flowcontrol -count=1 >/dev/null)

real	0m5.252s
user	0m6.884s
sys	0m2.837s
```

with logging:
```
time $(go test -race -v k8s.io/apiserver/pkg/util/flowcontrol -count=1 >/dev/null)

real	0m18.145s
user	0m29.135s
sys	0m6.430s
```

With the configuration used in presubmits/periodic testing, this package alone produces ~115MB of logs:

```
go test -race -v k8s.io/apiserver/pkg/util/flowcontrol -count=1 | wc -c

122363748
```

the logs seem useful for debugging, but I'd suggest a package-local `var debug = false` var and making all the Log statements in the inner loops conditional on that var. When working on this package specifically, the debug logs could be enabled locally.

Reply:
great catch! done.

it's even more extreme in my machine:

testDebugLogs=true:

```
ok      k8s.io/apiserver/pkg/util/flowcontrol   14.088s                                                                                                                                                                                        
```

testDebugLogs=false:

```
ok      k8s.io/apiserver/pkg/util/flowcontrol   2.155s
```

Reply:
```
$ go test -race . -count=1 -v | wc -c
49820
```

Reply:
good start, looks like there are frequently logged lines in match_test as well:

```
go test -v k8s.io/apiserver/pkg/util/flowcontrol -race  -count=1 | egrep -o '^.+.go:\d+:' | sort | uniq -c
 604     match_test.go:49:
 300     match_test.go:69:
```

Reply:
I force pushed just before your comment :)